packages/oidc-key-registry-server/src/fetchers/google.ts

```diff
@@ -0,0 +1,25 @@
+import type { Key, KeyFetcher } from "../types";
+
+export class GoogleFetcher implements KeyFetcher {
+  private apiUrl = "https://www.googleapis.com/oauth2/v3/certs";
+
+  async fetchKeys(): Promise<Key[]> {
+    const response = await fetch(this.apiUrl);
+    if (!response.ok) throw new Error(`Google API error: ${response.status}`);
+
+    const data = await response.json();
+    return data.keys.map((key: any) => ({
+      kid: this.toBytes32(key.kid),
+      n: this.toBytes(key.n),
+      e: this.toBytes(key.e),
+    }));
+  }
+
+  toBytes32(str: string): string {
+    return `0x${str.padStart(64, '0')}`;
+  }
+
+  toBytes(str: string): string {
```

Comment:
Can we call this "toHex"? If I read "toBytes" I expect a Buffer or a Uint8Array as return value.